__tests__/i18n.test.ts

```diff
@@ -0,0 +1,37 @@
+import { isObject } from '@klasa/utils';
+import { LanguageKeys } from '@lib/types/Languages';
+import { createLanguageStore } from '@mocks/MockLanguageStore';
+import English from '@root/languages/en-US';
+import Spanish from '@root/languages/es-ES';
+import { client } from '@testlib/MockInstances';
+
+/**
+ * Convert an object to a tuple.
+ * @description Adopted from `@klasa/utils`'s `objecToTuples`, but without including values which will never match between languages
+ * @param value The object to convert
+ * @param prefix The prefix for the key
+ */
+export default function objectToTuples<T = Record<string, unknown>>(original: T, prefix = ''): string[] {
```

Comment:
```suggestion
function objectToTuples<T = Record<string, unknown>>(original: T, prefix = ''): string[] {
```

Woops, that was a copypasta mistake